recipes/pangeo-forge-recipes-feedstock/meta.yaml

```
@@ -0,0 +1,66 @@
+{% set name = "pangeo-forge-recipes" %}
+{% set version = "0.8.2" %}
+{% set hash_val = "63512b9fe2898332a6293e60bb36e7c3e5e2643cc52767e3622a051a738c154f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash_val }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.8
+    - dask
+    - distributed
+    - fsspec[http] >=2021.6.0
```

```suggestion
    - fsspec >=2021.6.0
```

@martindurant - can you confirm that this is right for specifying the conda fsspec dependencies?

---

Correct, conda doesn't understand the `[...]` selector thing. You probably want to ensure aiohttp and requests are explicitly included in the list.

---

Will fsspec really not bring those along with it?

---

No, the fsspec conda package has no dependencies at all, since some backends just work. We could consider splitting out an fsspec-base package and a more complete fsspec with more requirements, but it would be hard to decide where to draw the line of what to include.

---

@xhochy  thanks a lot for your review here. I think my suggestions updates the dependencies correctly.

cisaacstern - the author of this PR - is on vacation today and tomorrow. I have no way to push to the branch on his fork. We would really like to get this PR merged asap. Do you as a maintainer have the ability to accept the suggestion? If so, we would 🙏  sincerely appreciate your help.